shared/src/lib.rs

```diff
@@ -16,8 +16,11 @@ pub mod store;
 pub mod tx_pool;
 mod tx_proposal_table;
 
-#[cfg(test)]
-mod tests;
+// These tests are from testnet data dump, they are hard to maintenance.
+// Although they pass curruent unit tests, we still comment out here,
+// Keep the code for future unit test refactoring reference.
+// #[cfg(test)]
+// mod tests;
```

Comment:
I think we still need these tests. some problem is hard to reproduce with unit tests, we need to maintain testnet/mainnet data that once broken network.